Dockerfile

```diff
@@ -24,35 +24,78 @@ RUN curl --location https://github.com/marbl/Mash/releases/download/v2.2/mash-Li
     rm -r mash*
 
 
-RUN apt-get install --yes gcc && \
-    apt-get install --yes --reinstall zlibc zlib1g zlib1g-dev
+RUN apt-get install --yes gcc=4:6.3.0-4 && \
+    apt-get install --yes --reinstall zlibc=0.9k-4.3 zlib1g=1:1.2.8.dfsg-5 zlib1g-dev=1:1.2.8.dfsg-5
 
+# dRep says v2.6.3 (2016) confirmed works, on its wiki
 RUN git clone https://github.com/hyattpd/Prodigal && \
```

Comment:
Can you instead just download a specific release from this page? https://github.com/hyattpd/Prodigal/releases
It might be more reliable than git commands